### PR TITLE
Blog cleanup tasks

### PR DESCRIPTION
**Pull request by hauten**
- Revised tone of blog posts to be less formal
- Renamed one blog post .md file
- Made a few editorial fixes (capitalization, etc.)
- Updated a couple of links
- Added more tags to posts
- Updated events.md to match

### CODE REVIEW

File: _posts/2019-12-05-spack-wins-rd-100-award.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ date: 2019-12-05 00:02:00
...
 
-The trade journal *R&D World Magazine* announced the winners of its annual awards, sometimes called the “Oscars of invention,” on October 31. The awards ceremony took place on December 5. The competition recognizes top scientific and engineering products and technologies with commercial potential. Spack was one of [four LLNL technologies](https://www.llnl.gov/news/llnl-researchers-win-four-rd-100-awards) that won. Spack also received a silver medal in the R&D 100 Awards’ Market Disruptor category. Check out the [video](https://youtu.be/D0p5xpsboK4) we submitted for the award.
+The trade journal *R&D World Magazine* announced the winners of its annual awards, sometimes called the “Oscars of invention,” on October 31. And Spack is one of [the winners](https://www.rdworldonline.com/2019-rd-100-award-winners-unveiled/)! The competition recognizes top scientific and engineering products and tech with commercial potential. We also received a silver medal in the Market Disruptor category. Check out [the video](https://youtu.be/D0p5xpsboK4) we submitted for the award and [this photo](https://twitter.com/Livermore_Comp/status/1207092109297123328) of Greg Becker juggling the plaques.
```
Comment (tgamblin) [inline _posts/2019-12-05-spack-wins-rd-100-award.md]: can we remove "the Oscars of Invention"?  The only place it's call that is on the R&D 100 site and in their public announcements 🤨.

**Review by tgamblin — APPROVED** [review]

**Review by tgamblin — CHANGES_REQUESTED** [review]
forgot to hit request changes for the one minor request above.

File: _posts/2019-12-05-spack-wins-rd-100-award.md (hunk 1 above)
Comment (hauten) [reply _posts/2019-12-05-spack-wins-rd-100-award.md]: ok, added commit below

**Review by tgamblin — APPROVED** [review]
@hauten: this looks great!  Thanks for updating the posts!

**Pull request merged**